software/wmla120_ansible/anaconda_prep_x86_64.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
After the 1.2 release is out I'd like to try to get most (perhaps all) of these playbooks merged with the Power versions

Reply:
agree. will introduce an issue for this.